src/olympia/constants/search.py

```diff
@@ -22,6 +22,7 @@
     'german': ['de'],
     'greek': ['el'],
     'english': ['en-us', 'en-ca', 'en-gb'],
+    'punjabi': ['pa-IN'],
```

Comment:
Don't do this. Elasticsearch does not have a `punjabi` analyzer.

Reply:
Thanks, removed it , lets see how does travis play now :)